packages/gatsby-source-contentful/src/normalize.js

```diff
@@ -2,14 +2,13 @@ const _ = require(`lodash`)
 const crypto = require(`crypto`)
 const stringify = require(`json-stringify-safe`)
 const deepMap = require(`deep-map`)
+const { makeTypeName, makeTextNodeType } = require(`./types`)
 
 const digest = str =>
   crypto
     .createHash(`md5`)
     .update(str)
     .digest(`hex`)
```

Comment:
new code style suggestion from:

#8805 Issue: feat: update gatsby plugins to use new createContentDigest helper

```suggestion
const digest = str => createContentDigest(str)
```
and add to imports:
```javascript
import { createContentDigest } from "gatsby-core-utils"
```

or better replace all usages of `digest` with `createContentDigest`